### PR TITLE
Fix spelling, grammar and style

### PR DESCRIPTION
**Pull request by michalfabik**
Signed-off-by: Michal Fabik <mfabik@redhat.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,45 +5,45 @@ This is the source code of ABRT's blog. You can visit it on
...
     title:  "Title of your post"
     date:   2017-01-22 18:48:44 +0100
-    author: your_name
+    author: <Your Name>
     visible: 1
-    categories: devided by space
+    categories: <space-separated list>
     ---
-    Here put your article in markdown
+    Put your article in markdown format here.
```
Comment (mgrabovsky) [inline README.md]: If we want to be really pedantic, Markdown is spelled with the initial letter in uppercase.

Comment (michalfabik) [reply README.md]: Thanks, fixed.

**Comment by mgrabovsky** [comment]
Looks all good. Thanks.

File: README.md (hunk 1 above)
Comment (ernestask) [reply README.md]: Do we ever not want to be pedantic?

**Pull request merged**